torch/distributed/rpc/__init__.py

```diff
@@ -50,29 +51,51 @@ def init_model_parallel(
                         ``Worker1``) Name can only contain number, alphabet,
                         underscore, and/or dash, and must be shorter than
                         128 characters.
-            self_rank (int): a globally unique id/rank of this node.
             init_method(str): backend specific init arguments.
-            num_send_recv_threads(int): Number of threads for send/recv work.
-            rpc_timeout (datetime.timedelta): Timeout for RPCs. Defaults to 10 seconds.
-                0 means infinity.
+            self_rank (int): a globally unique id/rank of this node.
+            world_size (int): The number of workers in the group.
```

Comment:
adding one line for `rpc_agent_options`?

Reply:
Added.